js-pkg/client/src/provider.ts

```diff
@@ -413,6 +501,7 @@ export class YSweetProvider {
   private websocketError(event: Event) {
     this.emit(EVENT_CONNECTION_ERROR, event)
     this.setStatus(STATUS_ERROR)
+    this.clearHeartbeat()
```

Comment:
Ditto re: clearing connection timeout
```suggestion
    this.clearHeartbeat()
    this.clearConnectionTimeout()
```